packages/next/build/webpack/plugins/build-manifest-plugin.ts

```diff
@@ -166,19 +150,6 @@ export default class BuildManifestPlugin {
               false
             )};` + `self.__BUILD_MANIFEST_CB && self.__BUILD_MANIFEST_CB()`
           )
-
-          if (this.modern) {
-            const modernClientManifestPath = `${CLIENT_STATIC_FILES_PATH}/${
-              this.buildId
-            }/_buildManifest.module.js`
-
-            compilation.assets[modernClientManifestPath] = new RawSource(
```

Comment:
We still need to generate this bundle, correct? Can you explain why this is removed?